README.md

```diff
@@ -16,10 +16,36 @@ The following AWS Config Rules are supported:
 
 ## Usage
 
-    module "aws_config" {
-      source             = "trussworks/config/aws"
-      config_logs_bucket = "my-aws-logs"
-    }
+```terraform
```

Comment:
This is populated by `terraform-docs`  in https://github.com/trussworks/terraform-aws-config/blob/master/main.tf#L1

Reply:
Sorry, I don't think I was clear in the original review. The README is generated using a pre-commit hook, that runs something called `terraform-docs`. You'll want to move this usage update to https://github.com/trussworks/terraform-aws-config/blob/master/main.tf#L1  and run the pre-commit hooks. You'll need to install [pre-commit ](https://pre-commit.com/#install) if you don't already have it.

```
pre-commit install --install-hooks
pre-commit run --all-files
```

Reply:
I see what you mean - you may want to add a CONTRIBUTING file to this repo in the future to explaining this sort of stuff :)

Reply:
should be sorted now